tests/Zilliqa/test_zilliqa_local.py

```diff
@@ -261,6 +261,9 @@ def run_start(numdsnodes):
 		shutil.copyfile('ds_whitelist.xml', LOCAL_RUN_FOLDER + testfolders_list[x] + '/ds_whitelist.xml')
 		shutil.copyfile('shard_whitelist.xml', LOCAL_RUN_FOLDER + testfolders_list[x] + '/shard_whitelist.xml')
 		shutil.copyfile('constants_local.xml', LOCAL_RUN_FOLDER + testfolders_list[x] + '/constants.xml')
+		ipc_path = "/tmp/zilliqa.sock." + str(NODE_LISTEN_PORT + x)
```

Comment:
will this be better?
```
"/tmp/zilliqa"  + str(NODE_LISTEN_PORT + x) + ".sock"
```

Reply:
Sure. Did you actually meant to change it to 
`"/tmp/zilliqa.”  + str(NODE_LISTEN_PORT + x) + ".sock"`
Additional `.` after `zilliqa`

Reply:
Not too familar with unix domain sockets convention. But I was thinking something like `zilliqa5001.sock`